Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabPublicHandler.cpp

```diff
@@ -913,15 +914,20 @@ namespace AzToolsFramework
                     }
                     else
                     {
-                        PrefabDom newPatch;
-                        m_instanceToTemplateInterface->GeneratePatch(newPatch, beforeState, afterState);
-                        LinkId linkId = m_prefabFocusInterface->AppendPathFromFocusedInstanceToPatchPaths(newPatch, entityId);
-
-                        Internal_HandleContainerOverride(parentUndoBatch, entityId, newPatch, linkId);
```

Comment:
Lol, so the name of the function before was 'handleContainerOverride' for all type of overrides. Looks like we never changed it when override capability is expanded.

Reply:
Yeah. I'd like to refactor/rework this function in a follow-up PR. The code/workflow needs to be improved to clear confusion and there should be more documentation.

Reply:
Basically when we're override editing, and isNewParentOwnedByDifferentInstance is false, it doesn't seem like we need beforeState and afterState.